src/puppetlabs/puppetdb/http.clj

```diff
@@ -190,9 +190,11 @@
 (defn stream-json
   "Serializes the supplied sequence to `buffer`, which is a `Writer`
   object."
-  [coll buffer]
+  [jsonable buffer pretty-print]
```

Comment:
jsonable sounds like a boolean param to me... I only know it's the thing to be converted because I see it used to be called `coll`. If your intent is that it's no longer just a collection, the docstring needs to be updated as well.